test/parallel/test-http-client-default-headers-exist.js

```diff
@@ -47,8 +47,8 @@ const server = http.createServer(function(req, res) {
   const requestHeaders = Object.keys(req.headers);
   requestHeaders.forEach(function(header) {
     assert.notStrictEqual(
-      expectedHeaders[req.method].indexOf(header.toLowerCase()),
-      -1,
+      expectedHeaders[req.method].includes(header.toLowerCase()),
+      false,
```

Comment:
We have [`assert.fail()`](https://nodejs.org/dist/latest-v6.x/docs/api/assert.html#assert_assert_fail_actual_expected_message_operator) to test if something is falsy, but it's not a strict comparison (it's `==` not `===`), and I find the `assertStrictEqual(..., false)` more clear (not sure why we have `assert.fail` not `assert.false`).